console-subscriber/src/visitors.rs

```diff
@@ -36,6 +43,30 @@ pub(crate) struct FieldVisitor {
     meta_id: proto::MetaId,
 }
 
+/// Used to extract the fields needed to construct
+/// an Event::Spawn from the metadata of a tracing span
+/// that has the following shape:
+///
+/// tracing::trace_span!(
+///     target: "tokio::task",
+///     "runtime.spawn",
+///     kind = "local",
+///     task.name = "some_name",
+///     loc.file = "some_file.rs"
+///     loc.line = 555,
+///     loc.col = 5,
+/// );
+///
+/// Fields:
+/// concrete_type - indicates the concrete rust type for this resource
+/// kind - indicates the type of resource (i.e. timer, sync, io )
```

Comment:
nit: let's add code formatting to this doc comment.

also, I don't think the list of fields are correct; it looks like it was copied from the docs for `ResourceVisitor`. i went ahead and updated that too:

```suggestion
/// Used to extract the fields needed to construct
/// an `Event::Spawn` from the metadata of a tracing span
/// that has the following shape:
///
/// ```
/// tracing::trace_span!(
///     target: "tokio::task",
///     "runtime.spawn",
///     kind = "local",
///     task.name = "some_name",
///     loc.file = "some_file.rs"
///     loc.line = 555,
///     loc.col = 5,
/// );
/// ```
///
/// # Fields
///
/// This visitor has special behavior for `loc.line`, `loc.file`, and `loc.col`
/// fields, which are interpreted as a Rust source code location where the task
/// was spawned, if they are present. Other fields are recorded as arbitrary
/// key-value pairs.
```